ui/shared/SplitViewHandle.qml

```diff
@@ -0,0 +1,10 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import "../imports"
+
+Rectangle {
+    // FIXME setting this 1 is prettier, but makes the handle complety unusable (too small to grab)
+    implicitWidth: 2
```

Comment:
I can still grab it at 1.. the width 2 just makes it too ugly 

if it's really ungrababble for you (like maybe in linux it's different), then a possible solution is to increase the width to 2 on mouse over

Reply:
I tried, it doesn't work. The width can be increased on Hover, but not the `implicitWidth` , which is the important property for grabbing it

Reply:
BTW, `1` worked well, **before** updating the version of the import. I think in QT 2, they changed how the handle works and made it worse